www/plugins/gatsby-remark-use-jsx/index.js

```diff
@@ -0,0 +1,22 @@
+const visit = require('unist-util-visit');
+
+// Always treat JS blocks as JSX.
+// TODO: maybe we can just change it in Markdown in the future?
+module.exports = ({markdownAST}) => {
+  visit(markdownAST, `code`, node => {
+    if (typeof node.lang !== 'string') {
+      return;
+    }
+    if (node.lang.indexOf('jsx') === 0) {
+      // Already JSX (with optional line range).
+      return;
+    }
+    // Turn JS into JSX, preserving the optional line range.
+    if (node.lang.indexOf('js') === 0) {
```

Comment:
Using ASTExplorer to quickly test, it seems possible for someone to specify multiple languages, via:
> ```js something-else

But I don't think we actually do this anywhere, do we? (Just curious why we're doing `indexOf` and `substring` instead of just string comparison)

Reply:
We're not specifying multiple languages, but we do use the `js{3,4,10-12}` trick for line ranges.

Reply:
Oh good point! I'd totally forgotten about our line highlighting syntax.